scripts/complexity.rb

```diff
@@ -0,0 +1,100 @@
+# @author Anton Hößl <e1427045@student.tuwien.ac.at>
+# @brief This script approximates the complexity of a elektra configuration namespace
+# @date 18.04.2018
+# @tags generator
```

Comment:
why generator?

Reply:
You are not bound to the tags which currently exist. Feel free to use any tag you find useful.

E.g. "specification" and "calculation" might be useful tags.